source/daplink/interface/main.c

```diff
@@ -57,13 +57,17 @@
 #define FLAGS_MAIN_CDC_EVENT    (1 << 11)
 // Used by msd when flashing a new binary
 #define FLAGS_LED_BLINK_30MS    (1 << 6)
+
 // Timing constants (in 90mS ticks)
-// USB busy time
+// USB busy time (~3 sec)
 #define USB_BUSY_TIME           (33)
-// Delay before a USB device connect may occur
+// Delay before a USB device connect may occur (~1 sec)
 #define USB_CONNECT_DELAY       (11)
+// Timeout for USB being configured (~5 sec)
```

Comment:
Any reason 5 seconds was chosen? This seems like a fairly long time. Could this delay be made shorter? Would ~1s be too short?

Reply:
5 seconds just sounded reasonable at the time between me and Melissa Hunter (apps engineer). :) A longer timeout gives a pretty good indication that there really isn't a USB host, which was my primary concern. But I can certainly make it shorter. Maybe 2 or 2.5 seconds? That seems short enough that a user wouldn't notice.

Reply:
2 or 2.5 seconds sounds good. I'll try it out this week to get a feel for how noticeable this is.

Reply:
Ok, thanks.